admin/src/app/home/orders/stock-orders/generated/generated.component.ts

```diff
@@ -500,6 +497,71 @@ export class GeneratedComponent implements OnInit, OnDestroy {
     }
   }
 
+  removeItemFromList(lineItems, list, count , success) {
+    let popedItem;
+    const index = list.findIndex((item) => item.id === lineItems.id);
+    if (index > -1) {
+      popedItem = list[index];
+      list.splice(index, 1)
+      if (list.length === 0 &&  count > 0){
+        this.reloadData();
+      }
+      success();
+    }
+    console.log(popedItem);
+    return popedItem;
+  }
+
+  addItemToList(lineItems, list, success) {
+    list.push(lineItems);
+    success();
+  }
+
+  moveItemToAnotherList(lineItems, data) {
```

Comment:
check if this is an instant action on staging, or should you spend time in setting a loader for this. is there a possibility that the user clicks on button twice coz this function is slow?

Reply:
The api loading stops after the item has moved